charts/fluid/fluid/templates/controller/dataset_controller.yaml

```diff
@@ -37,6 +37,10 @@ spec:
       #hostNetwork: true
       containers:
       - image: "{{ .Values.dataset.controller.image }}"
+        {{- with .Values.image.imagePullSecrets }}
```

Comment:
Could you please update the changelog of helm chart fluid?

Reply:
done. Thank you